jest.config.js

```diff
@@ -1,9 +1,12 @@
 module.exports = {
   testEnvironment: 'jsdom',
   transform: {
-    '^.+\\.(t|j)sx?$': ['@swc/jest']
+    '^.+\\.(t|j)sx?$': ['@swc/jest'],
   },
   moduleNameMapper: {
-    '\\.css$': 'identity-obj-proxy'
-  }
+    '\\.css$': 'identity-obj-proxy',
+    'react-i18next': '<rootDir>/test/__mocks__/react-i18next.ts',
+  },
+  modulePathIgnorePatterns: ['.next'],
+  setupFilesAfterEnv: ['<rootDir>/test/set-up-tests.js'],
```

Comment:
[nit] [non-blocking] very very minor, but grammatically, setup is used as a noun, rather than verb (set up) - you may wish to adopt the same convention.

Reply:
Ohh tbh I did notice that I did it differently but it's because when I made the file, I was reading it like 'setting up things for tests before you run them', not a test setup.